src/controllers/controller.line.js

```diff
@@ -177,38 +175,6 @@ module.exports = DatasetController.extend({
 		return values;
 	},
 
-	calculatePointY: function(value, index, datasetIndex) {
```

Comment:
For backwards compatibility, maybe we should leave this method even if it's unused, but deprecate it and add a todo to remove in v3

Reply:
done